api/pom.xml

```diff
@@ -18,9 +15,41 @@
     </parent>
 
     <name>Vitro API</name>
-
     <build>
         <plugins>
+            <plugin>
```

Comment:
General note: please remove changes not specific to this PR. There are a lot of modifications from other PR that aren't necessary here.